### DISCUSSION TITLE
إعتذار

### DISCUSSION THREAD

**Discussion by MutlaqAldhbuiub** [sudogroup/SudoBot#51] [3-بوح الشعر والقصائد]
في نهاية اليوم، أشكر البحر لإبعاد كل الطاقه السلبية التي بداخلي، وكذلك رؤية أخي 
@sudomaze 
@Mo9a7i 
.
.
وبالنسبة للعالم البرمجي 🖥️:
1. أنتهيت من إجتماع عمل و مازن سفل فيني بالمعلومات وإظهار التقدم مع 
@sudomaze

2. قمت بإعداد جدول بسيط لتنظيم وتنسيق تسفيل مازن الخاص لإنهاء المشاريع الحالية، وتعلم بعض المهارات الجديدة

علما بأن غدآ سيكون يوم مليء بالعمل الجاد بمشاركة أعضاء فريق شفتات 👨‍💻

إنتهى

**Comment by mo9a7i** [comment]
> في نهاية اليوم، أشكر البحر لإبعاد كل الطاقه السلبية التي بداخلي، وكذلك رؤية أخي 
> 
> @sudomaze 
> 
> @Mo9a7i 
> 
> .
> 
> .
> 
> وبالنسبة للعالم البرمجي 🖥️:
> 
> 1. أنتهيت من إجتماع عمل و مازن سفل فيني بالمعلومات وإظهار التقدم مع 
> 
> @sudomaze
> 
> 
> 
> 2. قمت بإعداد جدول بسيط لتنظيم وتنسيق تسفيل مازن الخاص لإنهاء المشاريع الحالية، وتعلم بعض المهارات الجديدة
> 
> 
> 
> علما بأن غدآ سيكون يوم مليء بالعمل الجاد بمشاركة أعضاء فريق شفتات 👨‍💻
> 
> 
> 
> إنتهى

لم يكن بالإمكان اكثر مما كان، ولقد حقت دانة الصفر وامتدت عروق الضمأ وانتشر الورد في الوريد واشتد الوعيد

اما بعد

أحبتي اعضاء منتدى سودوميز
في فترات الشتاء الماضية، انبتت بوادر الامل في قت هب، شمعة ابداعٍ تجلى نورها في انحاء المعمورة

وانه لمن الممكن ان يستمر هذا الدأب الجاد في إستدراج تلك الاكواد المتلثمة بدم خاينها

لقد اسعدني ان ارى هذا الصرح من المشاركات الوجدانية وسلامتكم
